packages/grub2/collection.yaml

```diff
@@ -1,7 +1,21 @@
 packages:
-  - name: "grub2-artifacts"
-    category: "system"
-    version: 0.0.2+1
-  - name: "grub2-config"
-    category: "system"
-    version: 0.0.11+1
+- name: "grub2-artifacts"
+  category: "system"
+  version: "0.0.3"
+  provides:
+  - name: "grub-artifacts"
+    version: ">0.0.2"
+
+- name: "grub2-config"
+  category: "system"
+  version: "0.0.13"
+  provides:
+  - name: "grub-config"
+    version: ">0.0.12"
```

Comment:
I think this will clash if anyone is consuming grub-config 0.0.10 for example no? should we just `version: ">=0"` and call it a day?

Reply:
I am not so sure, since this is 0.0.13 and 0.0.10 was provided by another package definition, most likely providing, the same, but still a different package. So if someone call for grub-config 0.0.10 probably I prefer to make explicit they require the old build.